developers.adoc

```diff
@@ -225,25 +225,20 @@ to `X.Y.Z` format (e.g. `0.9.0` for 0.9 release)
 
 === Python
 
-* Create a `~/.pypirc` file with this (https://github.com/pypa/setuptools/issues/941#issuecomment-279123202[source]):
-+
-[source,ini]
-----
-[distutils]
-index-servers =
-    pypi
-
-[pypi]
-repository: https://pypi.python.org/pypi
-username: <login>
-password: <pass>
-----
-* Pump version in `kaitaistruct.py`, seek `__version__ =`
-* `python3 setup.py sdist upload`
-** (use `python3 setup.py sdist upload -r pypitest` to publish to testing server)
-* Check that new version appears at https://pypi.python.org/pypi/kaitaistruct[https://pypi.python.org/pypi/kaitaistruct]`/$VERSION`
+* Make sure that the needed Python packaging tools are installed and up-to-date:
+** Debian/Ubuntu/etc.: `sudo apt install python3-setuptools python3-wheel twine`
+** Other Linuxes: see https://packaging.python.org/guides/installing-using-linux-tools/[Python Packaging User Guide]
+** Everywhere else: use your system package manager, or `python3 -m pip install --upgrade setuptools twine wheel`
```

Comment:
Doesn't `python -m pip install ...` work everywhere? I think there's no need to distinguish operating systems and Linux distributions.

Reply:
It does, sort of. But when you have Python installed through a package manager (which is common on Linux), it's expected that you install Python packages using the package manager and not `pip`. If you use `pip` anyway, you get at best a confusing situation with multiple versions of the same package installed, or at worst you break the installation managed by the package manager.

I'll rewrite this to be a bit shorter and more general.